app/Providers/RouteServiceProvider.php

```diff
@@ -52,7 +52,7 @@ public function map()
     protected function mapWebRoutes()
     {
         Route::middleware('web')
-             ->namespace($this->namespace)
+            //  ->namespace($this->namespace)
```

Comment:
The complete deletion would be better. Also, please remove the similar line on the `mapApiRoutes()` method, please.
```suggestion
```